tvtk/pyface/tvtk_scene.py

```diff
@@ -808,7 +816,7 @@ def _create_control(self, parent):
 
         self._interactor.initialize()
         self._interactor.render()
-        self.light_manager = light_manager.LightManager(self)
+        self.light_manager = light_manager.LightManager(weakref.proxy(self))
```

Comment:
Actually, it looks that it is not necessary to use a weakref here. Since as long as the `TVTKScene.close()` is called then the `_closed_fired` method will make sure that the cycle is properly broken.